src/AppBundle/Factory/Creators/UserCreator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\AppBundle\Factory\Creators;
+
+use App\AppBundle\Factory\Interfaces\FactoryInterface;
+use App\Entity\User;
+
+class UserCreator implements FactoryInterface
+{
+    public function createObject()
+    {
+        return new User();
```

Comment:
wypuszczasz niedziałający obiekt usera, tj bez wypełnionych pól, nie na tym to polega. Ty masz wypuścić gotowy obiekt, zdatny do użytku. Być może to wcale nie wzorzec Factory, poszukaj dobrze jak tworzyć obiekty aby uniknąć setterów w kontrolerze.

Reply:
Acz to też mój błąd, bo zasugerowałem bardzo mocno fabrykę/metodę fabrykującą. Zobacz wzorzec Builder (budowniczy).